src/components/TheHeader.vue

```diff
@@ -1,26 +1,28 @@
 <template>
   <section>
-    <article class='bone-yard-header'>
+    <article class="bone-yard-header">
       <h1>Bone Yard</h1>
-      <img class='bone-yard-logo' alt='Bone Yard pawprint logo' src='https://images-na.ssl-images-amazon.com/images/I/31jVeYUkbhL._AC_SY550_.jpg'/>
+      <img
```

Comment:
I appreciate you changing up the spacing/format here.  Looks much better split up.